### DISCUSSION TITLE
Stability of  values of parameter set

### DISCUSSION THREAD

**Discussion by francmst** [pybamm-team/PyBaMM#2983] [General]
I want to use Pybamm to simulate a battery with user-defined constituting equations. I started using one of the existing files ([PyBaMM/5-half-cell-model.ipynb at develop · pybamm-team/PyBaMM · GitHub](https://github.com/pybamm-team/PyBaMM/blob/develop/examples/notebooks/Creating%20Models/5-half-cell-model.ipynb)) instead of implementing submodules ex-novo. Before making any modification to the equations, I started by changing some of the parameters (such as particle thickness from 100 to 50 um, or particle radius from 10 to 4 um). Most of the modification results in an error, such as: 

SolverError: Could not find consistent states: Could not find acceptable solution: .../casadi/core/rootfinder.cpp:280: rootfinder process failed. Set 'error_on_fail' option to false to ignore this error.
SolverError: Could not find acceptable solution: .../casadi/core/rootfinder.cpp:280: rootfinder process failed. Set 'error_on_fail' option to false to ignore this error.

My understanding is that the solver can not find a suitable starting point for the iteration. Is my understanding correct? What is a possible solution or way around?

**Comment by brosaplanella** [comment]
You might want to play with the [solver settings](https://github.com/pybamm-team/PyBaMM/blob/develop/examples/notebooks/Getting%20Started/Tutorial%208%20-%20Solver%20options.ipynb) or the [meshes](https://github.com/pybamm-team/PyBaMM/blob/develop/examples/notebooks/Getting%20Started/Tutorial%209%20-%20Changing%20the%20mesh.ipynb). However, bear in mind that changing the parameters is quite tricky and you might end up in unphysical situations.
